src/main/java/eu/europa/ted/efx/sdk1/EfxExpressionTranslator08.java

```diff
@@ -59,7 +62,7 @@
  * {@link EfxExpressionTranslator08} only focuses on EFX expressions. To translate EFX templates
  * you need to use the {@link EfxTemplateTranslator08} which derives from this class.
  */
-@VersionDependentComponent(versions = {"0.8"}, componentType = VersionDependentComponentType.EFX_EXPRESSION_TRANSLATOR)
+@VersionDependentComponent(versions = {"1.0"}, componentType = VersionDependentComponentType.EFX_EXPRESSION_TRANSLATOR)
 public class EfxExpressionTranslator08 extends EfxBaseListener
```

Comment:
This class should not be named EfxExpressionTranslator**08** anymore.
Same for the other classes *08 in this package

Reply:
Fixed